doc/test-suite.py

```diff
@@ -55,32 +54,31 @@
     path2 = args.output_dir.replace("\ ", " ")
     dir1 = sorted(os.listdir(path1))
     for item1 in dir1:
-        if not(os.path.isdir(os.path.join(path1, item1))):
+        if not (os.path.isdir(os.path.join(path1, item1))):
             continue
 
         # create the output directory if necessary
-        if not(os.path.isdir(os.path.join(path2, item1))):
+        if not (os.path.isdir(os.path.join(path2, item1))):
             os.mkdir(os.path.join(path2, item1))
 
         dir2 = sorted(os.listdir(os.path.join(path1, item1)))
         for item2 in dir2:
             # skip directories
-            if not(os.path.isfile(os.path.join(path1, item1, item2))):
+            if not (os.path.isfile(os.path.join(path1, item1, item2))):
                 continue
             # skip hidden files
             if item2.startswith('.'):
                 continue
 
-            if shortlist and not(os.path.join(item1, item2) in shortlist):
+            if shortlist and not (os.path.join(item1, item2) in shortlist):
                 continue
 
             # reset the options
             options = testOptions.copy()
 
             # filenames (input MEI/XML and output SVG)
             inputFile = os.path.join(path1, item1, item2)
-            options.update({"xmlIdSeed": int(hashlib.sha256(
```

Comment:
What is the improvement here?

Reply:
Using the built-in function instead of mimicking it in Python.